SignalServiceKit/src/Loki/Utilities/FriendRequestExpireJob.swift

```diff
@@ -0,0 +1,214 @@
+/*
+ This class is used for settings friend requests to expired
+ This is modelled after `OWSDisappearingMessagesJob`.
+ */
+
+@objc(OWSLokiFriendRequestExpireJob)
+public class FriendRequestExpireJob: NSObject {
```

Comment:
I usually use ` : ` (with a space in front) for inheritance and `:` (with no space in front) for type declaration